Add blogpost on Blooms Taxonomy

ocaisa opened the pull request:
@psteinb Additions  and comments welcome

ChristinaLK commented:
Oops, meant to say that this should stay open in case there's discussion about the HPC name requirements.  But people can still discuss + we can revert if needed.

ocaisa commented on about.md:
```diff
@@ -4,7 +4,51 @@ layout: base
+[carpentries]: https://carpentries.org/
+[carpentries-coc]: https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html
+[pre-workshop]: google.link
+[post-workshp]: google.link
```
Typo `[post-workshop]`

ChristinaLK replied:
I'll fix this when I add the google links -- good catch.

Commit:
Add blogpost on Blooms Taxonomy

Commit:
Add teaser

psteinb commented:
So I fork, your fork of carpentries.org?

ocaisa commented:
Hmm, let me make you a maintainer rather than overcomplicate (especially since we are about to move the repo). I will also try to break the link to carpentries.org since it is no longer so relevant

ocaisa commented:
@psteinb I think you should have the rights to edit this now

psteinb commented:
Yes, I do!

tkphd commented on _posts/2020/11/2020-11-12-hpccarpentry-lesson-development.md:
```diff
@@ -0,0 +1,37 @@
+tags: ["Community", "Lesson development"]
+---
+
+Together with CodeRefinery ([1]), we helped to organise
```
For these relative links, I believe you need both the link-text *and* the link alias. So, a preferable format for these would be `([1][1])` instead of `([1])`.

tkphd replied:
Ooh, nevermind! I learned something today!

tkphd reviewed (changes requested):
Links will probably not work.

psteinb committed:
taxonomy material

- split ealier version into 2 parts (one to serve as intro, second to serve as feedback)
- intro, exercise results, discussion added

psteinb commented:
@ocaisa I did my due with https://github.com/ocaisa/hpc-carpentry.org/pull/3/commits/3d9d4327bd9523427dac68041b68c34118a70904 

Please have a look. I didn't render this to be honest, so I am not sure about aesthetics. Feel free to take it from here.

Commit:
Tidy up and format

Commit:
Remove Reference heading since refs don't appear anyway

Commit:
Add link to Slack channel

Commit:
Tweak connect page
